kong/db/declarative/import.lua

```diff
@@ -522,6 +522,11 @@ do
     end
 
     ok, err = load_into_cache_with_events_no_lock(entities, meta, hash, hashes)
+
+    if ok and transaction_id then
+      ok, err = kong_shm:set("declarative:current-transaction-id", transaction_id)
```

Comment:
```suggestion
      ok, err = kong_shm:set("declarative:current_transaction_id", transaction_id)
```

We usually use underscore instead of dashes for shdict keys. E.g:

https://github.com/Kong/kong/blob/083ab25d26acddcc5632305d698c854d67951859/kong/constants.lua#L201C1-L204